packages/gatsby-plugin-sharp/src/process-file.js

```diff
@@ -116,10 +116,6 @@ exports.processFile = (file, transforms, options = {}) => {
           adaptiveFiltering: false,
           force: transformArgs.toFormat === `png`,
         })
-        .webp({
-          quality: transformArgs.webpQuality || transformArgs.quality,
-          force: transformArgs.toFormat === `webp`,
-        })
```

Comment:
I think this should be the other way around?

I've recently noticed that my output from processing with `sharp` without gatsby differed in quality and filesize for some reason. Came across your PR here, and now it makes sense that it's due to the double compression as you've found here, but imagemin is the culprit.

It reduces filesize from 16.5KiB to 15.6KiB, but introduces some notable artifacts, both would be running at quality of 75. When removing the `sharp` processing that you've done here, just with imagemin instead, the output filesize is 17.1KiB for my test file and the quality is not as good.

Reply:
Yes, you right. It should be other way around - I should have been deleted second conversion with imageminWebp. I fixed it in new commit.